Tails specific: no need to save or restore the seed, we are amnesic.

--- chroot.orig/etc/init.d/urandom	2012-09-24 10:05:13.385058073 +0200
+++ chroot/etc/init.d/urandom	2012-09-24 10:47:23.717869294 +0200
@@ -6,2 +6,2 @@
-# Default-Start:     S
+# Default-Start:
-# Default-Stop:      0 6
+# Default-Stop:
